src/matches.js

```diff
@@ -1,8 +1,18 @@
+function assertNotNullOrUndefined(matcher) {
+  if (matcher == null) {
+    throw new Error(
+      `It looks like ${matcher} was passed instead of a matcher. Did you do something like getByText("${matcher}")?`,
```

Comment:
Shouldn't this be without quotes?
```suggestion
      `It looks like ${matcher} was passed instead of a matcher. Did you do something like getByText(${matcher})?`,
```